swagger.json

```diff
@@ -568,25 +568,24 @@
                 }
             }
         },
-        "/fleet/drivers/{driver_id}": {
-            "parameters": [
+        "/fleet/drivers/{driver_id OR external_id}": {
+            "get": {
+              "parameters": [
                 {
                     "$ref": "#/parameters/accessTokenParam"
                 },
                 {
-                    "name": "driver_id",
+                    "name": "driver_id OR external_id",
                     "in": "path",
                     "required": true,
-                    "description": "ID of the driver.",
-                    "type": "integer",
-                    "format": "int64"
+                    "description": "ID of the driver (integer) or corresponding external ID (alphanumeric string).",
```

Comment:
We may want to provide a more detailed description of this parameter, since it is really a `oneOf(driverId, externalId)`.